src/cascadia/TerminalCore/ColorFix.hpp

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#pragma once
+
+// We probably want to include something else here, whatever gives us COLORREF and BYTE
+#include "../../terminal/input/terminalInput.hpp"
+
+struct ColorFix
+{
+public:
+    ColorFix(COLORREF color);
+
+    static double GetDeltaE(ColorFix x1, ColorFix x2);

```

Comment:
Looks like `GetDeltaE` is only used in `ColorFix.cpp`. You may be able to make this `private`